.github/workflows/python-builder.yml

```diff
@@ -182,15 +182,15 @@ jobs:
             Python ${{ env.VERSION }}
 
       - name: Upload release assets
-        uses: actions/github-script@v3
+        uses: actions/github-script@v6
```

Comment:
Maybe it's a dumb question, but why are we jumping from V3 to V6?

Reply:
Just to keep up with the times. In order to get the latest changes, features, fixes for bugs and security vulnerabilities (if any) and so on.